services/libs/tinybird/datasources/integrations.datasource

```diff
@@ -0,0 +1,13 @@
+SCHEMA >
+    `id` String `json:$.record.id`,
+    `platform` String `json:$.record.platform` DEFAULT '',
+    `status` String `json:$.record.platform` DEFAULT '',
```

Comment:
Is the JSON schema on this one correct? Shouldn't it be just a string instead of a `platform`? Or, in case we have specific statuses somewhere, perhaps it should reference that?